### PR TITLE
Feature #48812 : US 34.2 : Revue des fichiers CSV de l'ensemble des m…

### PR DESCRIPTION
**Pull request by bsu-axelor**
…odules (Standards) de l'application

### CODE REVIEW

File: axelor-crm/src/main/resources/i18n/messages_fr.csv [hunk 1]
```diff
@@ -10,7 +10,7 @@
 "<span class='label label-warning'>There is already a lead with this name.</span>","<span class='label label-warning'>Il y a déjà une piste avec ce nom.</span>",,
 "<span style=""height:21px; line-height: 21px; margin-top:18px; display:inline-block"">VS</span>",,,
 "A lost reason must be selected","Une raison de perte doit être selectionnée",,
-"Action",,,
+"Action","Action",,
```
Comment (gdu-axelor) [inline axelor-crm/src/main/resources/i18n/messages_fr.csv:13]: No need to translate

File: axelor-crm/src/main/resources/i18n/messages_fr.csv [hunk 2]
```diff
@@ -438,7 +438,7 @@
 "Reportings","Rapports",,
 "Reports","Rapports",,
 "Required participant","Participants requis",,
-"Roles",,,
+"Roles","Rôles",,
```
Comment (gdu-axelor) [inline axelor-crm/src/main/resources/i18n/messages_fr.csv:441]: No need to translate

File: axelor-crm/src/main/resources/i18n/messages_fr.csv [hunk 3]
```diff
@@ -513,7 +513,7 @@
 "Unassigned Opportunities","Opportunités non assignées",,
 "Unassigned opportunities","Opportunités non assignées",,
 "Upcoming events","Prochains Evènements",,
-"Urgent",,,
+"Urgent","Urgent",,
```
Comment (gdu-axelor) [inline axelor-crm/src/main/resources/i18n/messages_fr.csv:516]: No need to translate

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 4]
```diff
@@ -119,11 +119,11 @@
 "Company department","Service",,
 "Company fixed phone","N° de téléphone de la société",,
 "Company's credit card","CB de la société",,
-"Complete",,,
+"Complete","Compléter",,
 "Complete my Timesheet","Compléter ma feuille de temps",,
 "Complete my expense","Compléter ma note de frais",,
 "Complete my leave request","Compléter ma demande de congés",,
-"Completed",,,
+"Completed","Complété",,
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:126]: terminé ?

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 5]
```diff
@@ -157,13 +157,13 @@
...
 "Created by","Créé par",,
 "Created on","Créé le",,
-"DATE D'ENTREE",,,
-"DATE DE SORTIE",,,
-"DATE NAISS",,,
-"DEPT NAISS",,,
-"DPAE","DPAE",,
-"DPAE filters",,,
-"DPAEs","DPAEs",,
+"DATE D'ENTREE",,"Key in FR",
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:160]: No need to translate. It's API key

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 6]
```diff
@@ -172,7 +172,7 @@
 "Date difference should be maximam 31 days.","La différence entre dates doit être de maximum 31 jours.",,
 "Date of birth","Date de naissance",,
 "Date of hire","Date d'embauche",,
-"Dates",,,
+"Dates","Dates",,
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:175]: No need to translate.

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 7]
```diff
@@ -197,8 +197,8 @@
 "Duration(in day per day)","Durée (en jour par jour)",,
 "Duration(minutes by day)","Durée (minutes par jour)",,
 "E-MAIL",,,
-"ETAB",,,
-"ETRANGER",,,
+"ETAB",,"Key in FR",
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:200]: No need to translate. It's API key

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 8]
```diff
@@ -509,8 +509,8 @@
 "Kilometric log list","Compteur kilométrique",,
 "Kilometrice Allowance Param","Paramètre barème kilométrique",,
 "Kilometrice Allowance Params","Paramètres barème kilométrique",,
-"Kilometrice log",,,
-"LIEU",,,
+"Kilometrice log","Historique barème kilométrique",,
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:512]: Compteur kilométrique ?

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 9]
```diff
@@ -509,8 +509,8 @@
 "Kilometric log list","Compteur kilométrique",,
 "Kilometrice Allowance Param","Paramètre barème kilométrique",,
 "Kilometrice Allowance Params","Paramètres barème kilométrique",,
-"Kilometrice log",,,
-"LIEU",,,
+"Kilometrice log","Historique barème kilométrique",,
+"LIEU",,"Key in FR",
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:513]: No need to translate. It's API key

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 10]
```diff
@@ -553,9 +553,9 @@
 "List of employees","Liste des Employés",,
 "Log Time","Temps passé",,
 "Log Times","Temps passés",,
-"Log time",,,
+"Log time","Temps passé",,
 "Logged Time","Temps passé",,
-"Login",,,
+"Login","Identifiant",,
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:558]: No need to translate.

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 11]
```diff
@@ -591,7 +591,7 @@
 "LunchVoucher.total","TOTAL GÉNÉRAL",,
 "LunchVoucherCommand","CommandeTR",,
 "M","M",,
-"MATRICULE",,,
+"MATRICULE","MATRICULE","Key in FR",
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:594]: No need to translate. It's API key

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 12]
```diff
@@ -605,7 +605,7 @@
 "Mass export list","Liste des exports de masse",,
 "Maximum Condition","Condition maximale",,
 "Maximum value for hours exceeded (configure daily work hours in employee)","Valeur maximale d'heures dépassée (veuillez configurer les heures de travail journalier dans l'employé)",,
-"Messages",,,
+"Messages","Messages",,
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:608]: No need to translate

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 13]
```diff
@@ -632,10 +632,10 @@
 "My leave requests","Mes demandes de congés",,
 "My timesheet","Ma feuille de temps",,
 "My timesheets","Mes feuilles de temps",,
-"NOM",,,
-"NOM DE JEUNE FILLE",,,
-"NOM MARITAL",,,
-"NUMERO INSEE",,,
+"NOM",,"Key in FR",
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:635]: No need to translate. It's API key

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 14]
```diff
@@ -731,7 +731,7 @@
 "Planned hours","Heures planifiées",,
 "Planned time","Temps planifié",,
 "Planning line cannot be in two different days","Une ligne de planning ne peut être sur 2 jours différents",,
-"Plannings",,,
+"Plannings","Emplois du temps",,
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:734]: Not sure about this

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 15]
```diff
@@ -854,8 +854,8 @@
 "Return to draft status","Retourner à l'état brouillon",,
 "Round trip","Aller-retour",,
 "SILAE",,,
-"SMIC value",,,
-"Salary change",,,
+"SMIC value","Valeur du SMIC",,
+"Salary change","Change de salaire",,
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:858]: ????

File: axelor-human-resource/src/main/resources/i18n/messages_fr.csv [hunk 16]
```diff
@@ -1023,8 +1023,8 @@
 "User","Utilisateur",,
 "User creation","Création de l'utilisateur",,
 "Users","Utilisateurs",,
-"VILLE",,,
-"VOIE",,,
+"VILLE",,"Key in FR",
```
Comment (gdu-axelor) [inline axelor-human-resource/src/main/resources/i18n/messages_fr.csv:1026]: No need to translate. It's API key

File: axelor-maintenance/src/main/resources/i18n/messages_fr.csv [hunk 17]
```diff
@@ -1,54 +1,38 @@
...
+"Bill of material","Nomenclature",,
+"Bills of material","Nomenclature",,
 "Bills of materials","Nomenclatures",,
-"Cancel",,,
-"Cancel Reason",,,
-"Canceled",,,
-"Complete",,,
+"Cancel","Annuler",,
+"Canceled","Annulé",,
+"Complete","Compléter",,
```
Comment (gdu-axelor) [inline axelor-maintenance/src/main/resources/i18n/messages_fr.csv:14]: Terminer ?

File: axelor-maintenance/src/main/resources/i18n/messages_fr.csv [hunk 18]
```diff
@@ -62,55 +46,26 @@
 "ManufOrder.machine","Machine",,
 "ManufOrder.machineType","Machine Type",,
 "Manufacturing Order","Ordre de Fabrication",,
-"Manufacturing order",,,
+"Manufacturing order","Ordre de Fabrication",,
```
Comment (gdu-axelor) [inline axelor-maintenance/src/main/resources/i18n/messages_fr.csv:49]: Ordre de fabrication

File: axelor-maintenance/src/main/resources/i18n/messages_fr.csv [hunk 19]
```diff
@@ -62,55 +46,26 @@
 "ManufOrder.machine","Machine",,
 "ManufOrder.machineType","Machine Type",,
 "Manufacturing Order","Ordre de Fabrication",,
-"Manufacturing order",,,
+"Manufacturing order","Ordre de Fabrication",,
 "Manufacturing orders","Ordres de fabrication",,
-"Moves In/Out",,,
 "My Maintenance requests","Mes demandes de maintenance",,
-"Name",,,
+"Name","Nom",,
```
Comment (gdu-axelor) [inline axelor-maintenance/src/main/resources/i18n/messages_fr.csv:52]: No need to translate

File: axelor-maintenance/src/main/resources/i18n/messages_fr.csv [hunk 20]
```diff
@@ -62,55 +46,26 @@
...
-"Options",,,
-"Partial Finish",,,
-"Pause",,,
-"Permanent",,,
-"Permanent manufacturing orders",,,
-"Phases",,,
-"Plan",,,
-"Planned dates",,,
-"Pre-fill operations",,,
+"Plan","Plan",,
```
Comment (gdu-axelor) [inline axelor-maintenance/src/main/resources/i18n/messages_fr.csv:54]: Planifier? No need to translate (axelor-production)

File: axelor-maintenance/src/main/resources/i18n/messages_fr.csv [hunk 21]
```diff
@@ -62,55 +46,26 @@
...
 "Preventive","Préventive",,
-"Print",,,
-"Print Production Process",,,
-"ProdProcess.machine",,,
-"Product",,,
-"Production",,,
-"Production process",,,
+"Print","Imprimer",,
```
Comment (gdu-axelor) [inline axelor-maintenance/src/main/resources/i18n/messages_fr.csv:56]: No need to translate

File: axelor-maintenance/src/main/resources/i18n/messages_fr.csv [hunk 22]
```diff
@@ -62,55 +46,26 @@
...
+"ProdProcess.machine","Machine",,
```
Comment (gdu-axelor) [inline axelor-maintenance/src/main/resources/i18n/messages_fr.csv:57]: No need to translate (axelor-production)

File: axelor-maintenance/src/main/resources/i18n/messages_fr.csv [hunk 23]
```diff
@@ -62,55 +46,26 @@
...
+"Product","Produit",,
```
Comment (gdu-axelor) [inline axelor-maintenance/src/main/resources/i18n/messages_fr.csv:58]: No need to translate

File: axelor-maintenance/src/main/resources/i18n/messages_fr.csv [hunk 24]
```diff
@@ -62,55 +46,26 @@
...
+"Production process","Gamme",,
```
Comment (gdu-axelor) [inline axelor-maintenance/src/main/resources/i18n/messages_fr.csv:59]: No need to translate (axelor-production)

File: axelor-marketing/src/main/resources/i18n/messages_fr.csv [hunk 25]
```diff
@@ -19,22 +19,22 @@
 "Cannot generate targets. Please check lead filter of Target Model.","Impossible de générer les cibles. Vérifiez le filtre Piste du modèle cible",,
 "Cannot generate targets. Please check partner filter of Target Model.","Impossible de générer les cibles. Vérifiez le filtre Tiers du modèle cible",,
 "Clear Selection","Effacer la sélection",,
-"Configuration",,,
+"Configuration","Configuration",,
```
Comment (gdu-axelor) [inline axelor-marketing/src/main/resources/i18n/messages_fr.csv:22]: No need to translate

File: axelor-marketing/src/main/resources/i18n/messages_fr.csv [hunk 26]
```diff
@@ -19,22 +19,22 @@
...
 "Contact","Contact",,
 "Contact/Partner reminder template","Modèle de relance client",,
 "Contact/Partners template","Modèle Contact/Tiers",,
 "Days","Jours",,
 "Draft","Brouillon",,
 "Duration","Durée",,
 "Duration type","Type durée",,
-"Email",,,
+"Email","Email",,
```
Comment (gdu-axelor) [inline axelor-marketing/src/main/resources/i18n/messages_fr.csv:30]: No need to translate

File: axelor-production/src/main/resources/i18n/messages_fr.csv [hunk 27]
```diff
@@ -80,12 +80,12 @@
...
 "Change allocated qty","Changer la quantité allouée",,
 "Charge by machine","Charge par machine",,
-"Children MO","OF Enfants",,
+"Children MO",,,
 "Code","Code",,
 "Comments","Commentaires",,
 "Companies","Sociétés",,
 "Company","Société",,
-"Completed",,,
+"Completed","Complété",,
```
Comment (gdu-axelor) [inline axelor-production/src/main/resources/i18n/messages_fr.csv:88]: Terminé ?

File: axelor-production/src/main/resources/i18n/messages_fr.csv [hunk 28]
```diff
@@ -108,7 +108,7 @@
 "Configurator Phase","Configurateur de phase",,
 "Configurator Phases","Configurateur de phases",,
 "Configurator Prod Process","Configurateur de gamme",,
-"Configurator Prod Process filters",,,
+"Configurator Prod Process filters","Filtres Gamme pour configurateur",,
```
Comment (gdu-axelor) [inline axelor-production/src/main/resources/i18n/messages_fr.csv:111]: Filtres configurateur de gamme

**Review by gdu-axelor — CHANGES_REQUESTED** [review]

**Comment by ale-axelor** [comment]
To be reopened when updated with correct information (see related issue).

**Pull request closed without merging**